cluster/ci/config/prow/plugins.yaml

```diff
@@ -1127,6 +1127,10 @@ config_updater:
       name: ci-operator-master-configs
       additional_namespaces:
       - ci-stg
+    ci-operator/config/**/*master__variant.yaml:
```

Comment:
Is it possible to follow the format like ```ci-operator/config/**/*master__*.yaml``` so that anyone can have more control over choosing the variant file name. WDYT ?